packages/db/src/models/granule-execution-history.ts

```diff
@@ -0,0 +1,50 @@
+import Knex from 'knex';
+
+import { isRecordDefined } from '../database';
+import { tableNames } from '../tables';
+
+import { PostgresGranuleExecution } from '../types/granule-execution-history';
+
+export default class GranuleExecutionHistoryPgModel {
```

Comment:
It seemed like having a model for just fetching records from the granuleExecutionHistory table (table linking granules to executions) was ideal, but feel free to challenge me on it